### DISCUSSION TITLE
Use a shared Google Calendar for team leave

### DISCUSSION THREAD

**Discussion by choldgraf** [2i2c-org/team-compass#82] [Ideas]
# Background

We want our team to be able to step away from work when they need to take a vacation, a sick day, or just a day or two to relax. We need a way to signal to other team members when this is happening, so that we know what to expect about one another's availability.

# Proposal

I suggest that we create a "Team Away Calendar" - this would be a calendar shared by anyone on the engineering team. Anyone on the team could add to it. When you plan on taking some time away from work, create a day-long event in the calendar with the title `AWAY: <your-name>`. If you wish, you could also add a reason for being away. For example `AWAY: Chris H, Vacation 🌴`.

Then, this calendar would become the official record of when team members were planning to be unavailable.

Do others think this would be a useful team practice to adopt?

checkout here for a PR to implement this: https://github.com/2i2c-org/team-compass/pull/96

**Comment by damianavila** [comment]
I have used that in the past and it worked (and continues working) pretty well, IMHO.

**Comment by yuvipanda** [comment]
Would this be in addition to a line in the team sync issue?

**Reply by choldgraf** [reply]
Yes - the team sync issue is for this week only, where the Google calendar would be for future times to be away as well

**Reply by yuvipanda** [reply]
Cool, +1. So for short-term availability (a few hours, a day here, etc) we'll just put it in team sync, and for longer periods farther away we'll put it in the calendar.

**Reply by choldgraf** [reply]
Yeah I don't think we wanna micromanage people's weekly availability on a calendar ("I'm gonna be away on Thursday for the afternoon") but I think we *do* want to give team members insight into planned a absences ("I'm gonna take next Thursday and Friday off" or "I'm going on vacation the first two weeks of july")

**Reply by choldgraf** [reply]
Hey all - checkout here for a PR to implement this: https://github.com/2i2c-org/team-compass/pull/96
